merge_values_test.go

```diff
@@ -142,7 +148,7 @@ func Test_MergeValues(t *testing.T) {
 				"another": []byte("test: val"),
 				"test":    []byte("test: val"),
 			},
-			errorMatcher: IsExecutionFailed,
+			errorMatcher: IsParsingSrcFailedError,
```

Comment:
If it's two key cm, I think we should indicate this is wrong to customers, by using app CRs. So this is fine.